lib/minio_runner/network.rb

```diff
@@ -41,6 +73,13 @@ def download(url, &block)
       ensure
         tempfile&.close!
       end
+
+      def log_time_error(request_start_time)
+        if (Time.now - request_start_time) > Network::LONG_RESPONSE_TIME_SECONDS &&
+             MinioRunner.config.minio_domain.end_with?(".local") && MinioRunner::System.mac?
+          MinioRunner.logger.warn(MAC_OS_LOCAL_DOMAIN_ERROR_MESSAGE.delete("\n"))
```

Comment:
Why the `.delete("\n")`? 😃

Reply:
Because the logger (which is going to `STDOUT`) prints `"\n" from `MAC_OS_LOCAL_DOMAIN_ERROR_MESSAGE` literally ... I am not sure how to get around this :sweat_smile:

Reply:
That's because of `#dump` used here: https://github.com/discourse/minio_runner/blob/b7ffce31a0aba2e7b11391698d334962fc0d5ec0/lib/minio_runner.rb#L45C49-L45C49 😛

Reply:
me rn @CvX :

![playedyourself](https://github.com/discourse/minio_runner/assets/920448/f3b2ecd5-efa7-419e-aced-73a507539745)